internal/domain/company.go

```diff
@@ -56,22 +61,69 @@ type GitHubProfile struct {
 type GlassdoorProfile struct {
 	OverviewURL string
 	ReviewsURL  string
+	Jobs        string // will be mostly outdated
+	Reviews     string
+	Salaries    string
+	ReviewsRate string
 	Verified    bool
 }
 
+type BlindProfile struct {
+	Alias       string
+	Employees   string
+	Salary      string
+	Reviews     string
+	ReviewsRate string
+}
+
+type LevelsFyiProfile struct {
+	Alias     string
+	Employees string
+}
+
+type IndeedProfile struct {
+	Alias string // for now, will be more in the future
+}
+
+type Vacancy struct {
+	Title       string
+	Description string
```

Comment:
"Description" -> "Proof that the vacancy is for a particular technology" -> "ShortDescriptionForProof"